ecs-agent/daemonimages/csidriver/util/fs/fs.go

```diff
@@ -27,25 +27,33 @@ type UsageInfo struct {
 
 // Info linux returns (available bytes, byte capacity, byte usage, total inodes, inodes free, inode usage, error)
 // for the filesystem that path resides upon.
-func Info(path string) (int64, int64, int64, int64, int64, int64, error) {
+func Info(path string) (
+	available int64, capacity int64, used int64,
+	totalInodes int64, freeInodes int64, usedInodes int64,
+	err error,
+) {
 	statfs := &unix.Statfs_t{}
-	err := unix.Statfs(path, statfs)
+	err = unix.Statfs(path, statfs)
 	if err != nil {
-		return 0, 0, 0, 0, 0, 0, err
+		available = 0
+		capacity = 0
+		used = 0
+
+		totalInodes = 0
+		freeInodes = 0
+		usedInodes = 0
+		return
```

Comment:
Why do we want to silently swallow the error instead of raising it?

Reply:
It won't be silent. We are using named return parameter, so we don't have to list all var in the return statement.

Reply:
meaning `err` won't be nil in this case

Reply:
I see. TIL.